modules/stats/tradingmodule.py

```diff
@@ -28,48 +28,74 @@ def __init__(self, config: TradingModuleConfig):
         self.closed_trades = []
         self.open_trades = []
         self.budget_per_timestamp = {}
+        self.open_trades_open_per_timestamp = {}
+        self.open_trades_low_per_timestamp = {}
         self.realised_profits = []
         self.total_fee_amount = 0
 
     def tick(self, ohlcv: dict, data_dict: dict) -> None:
         trade = self.find_open_trade(ohlcv['pair'])
         if trade:
             trade.update_stats(ohlcv)
-            self.open_trade_tick(ohlcv, trade)
+            closed = self.open_trade_tick(ohlcv, trade)
+            if not closed:
+                self.update_budget_per_timestamp(ohlcv)
         else:
             self.no_trade_tick(ohlcv, data_dict)
-        self.update_budget_per_timestamp_tracking(ohlcv)
+            self.update_budget_per_timestamp(ohlcv)
 
     def no_trade_tick(self, ohlcv: dict, data_dict: dict) -> None:
         if ohlcv['buy'] == 1:
             self.open_trade(ohlcv, data_dict)
 
-    def open_trade_tick(self, ohlcv: dict, trade: Trade) -> None:
+    def open_trade_tick(self, ohlcv: dict, trade: Trade) -> bool:
+        """
+        Method is called when specified pair has open trades.
+        checks for ROI
+        checks for stoploss
+        :param ohlcv: dictionary with OHLCV data for current tick
+        :type ohlcv: dict
+        :param trade: Trade corresponding to tick pair
+        :type trade: Trade
+        :return: None
+        :rtype: None
+        """
         stoploss_reached = self.check_stoploss_open_trade(trade, ohlcv)
         roi_reached = self.check_roi_open_trade(trade, ohlcv)
 
         if stoploss_reached and roi_reached:
-            trade.current = trade.open
-            trade.update_profits()
             self.close_trade(trade, reason=SellReason.STOPLOSS_AND_ROI, ohlcv=ohlcv)
         elif stoploss_reached:
             self.close_trade(trade, reason=SellReason.STOPLOSS, ohlcv=ohlcv)
         elif roi_reached:
             self.close_trade(trade, reason=SellReason.ROI, ohlcv=ohlcv)
         elif ohlcv['sell'] == 1:
             self.close_trade(trade, reason=SellReason.SELL_SIGNAL, ohlcv=ohlcv)
-        else:  # trade is not closed
-            self.update_value_per_timestamp_tracking(trade, ohlcv)
+        else:
+            self.update_open_trades_value_per_timestamp(trade, ohlcv)
+            return False
+        self.update_open_trades_value_per_timestamp(trade, ohlcv)
+        return True
 
     def close_trade(self, trade: Trade, reason: SellReason, ohlcv: dict) -> None:
         date = datetime.fromtimestamp(ohlcv['time'] / 1000)
         trade.close_trade(reason, date)
 
-        self.total_fee_amount += trade.close_fee_amount
+        if trade.sell_reason == SellReason.STOPLOSS_AND_ROI:
+            # Because trade had no impact on results, remove first issued fee from
+            # total amount of fee and reset trade stats.
+            first_fee = trade.starting_amount * trade.fee
+            self.total_fee_amount -= first_fee
```

Comment:
```suggestion
            self.total_fee_amount -= first_fee
```
            self.total_fee_paid -= first_fee